src/xcode/ENA/ENA/Source/Scenes/FriendsInvite/FriendsInviteController.swift

```diff
@@ -79,11 +79,15 @@ final class FriendsInviteController: UIViewController {
 
 extension FriendsInviteController: UIActivityItemSource {
 	func activityViewControllerPlaceholderItem(_: UIActivityViewController) -> Any {
-		shareTitle
+		return shareTitle
 	}
 
-	func activityViewController(_: UIActivityViewController, itemForActivityType _: UIActivity.ActivityType?) -> Any? {
-		shareUrl
+	func activityViewController(_: UIActivityViewController, itemForActivityType activityType: UIActivity.ActivityType?) -> Any? {
+		if activityType == .mail {
```

Comment:
what is different for mail here?

Reply:
The different is that we have the subject for mail (```activityViewController(_: UIActivityViewController, subjectForActivityType _: UIActivity.ActivityType?)```), and I didn't want to repeat here, since the same text is used in the subject.

Unfortunately some other mail clients, like Gmail, are not treated as ```mail```, so for them this doesn't apply, but it's good since Gmail takes this body text as the subject text too.

Please have a look at screenshots above.

Reply:
thanks for the explanation! :)